main.nf

```diff
@@ -252,8 +265,11 @@ process trim_galore {
     file "*_fastqc.{zip,html}" into trimgalore_fastqc_reports
 
     script:
+    tg_length = "--length ${params.length}"
+    c_r1 = ${params.clip_R1} > 0 ? "--clip_R1 ${params.clip_R1}" : ''
+    tpc_r1 = ${params.three_prime_clip_R1} > 0 ? "--three_prime_clip_R1 ${params.three_prime_clip_R1}" : ''
```

Comment:
`${}` is only required when in double quotes. So this should work:

```groovy
c_r1 = params.clip_R1 > 0 ? "--clip_R1 ${params.clip_R1}" : ''
tpc_r1 = params.three_prime_clip_R1 > 0 ? "--three_prime_clip_R1 ${params.three_prime_clip_R1}" : ''
```

Reply:
Thanks Phil! /chuan